sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb

```diff
@@ -194,7 +217,6 @@
     "STUDENT_MODEL_VERSION = 1\n",
```

Comment:
Would we want to pin to a version or may be pick the latest?

Reply:
I thought of this while writing, the pro of pinning to 1 is that we don't get latest changes. The con of is that breaking changes may be introduced in further versions. Decided to keep it as 1, let me know if you feel otherwise.